stl/inc/xutility

```diff
@@ -5997,8 +5997,12 @@ _NODISCARD _CONSTEXPR_BIT_CAST bool _Is_inf(const _Ty _Xx) { // constexpr isinf(
 // FUNCTION TEMPLATE _Is_finite
 template <class _Ty, enable_if_t<is_floating_point_v<_Ty>, int> = 0>
 _NODISCARD _CONSTEXPR_BIT_CAST bool _Is_finite(const _Ty _Xx) { // constexpr isfinite()
+#if defined(__LDBL_DIG__) && __LDBL_DIG__ == 18
+    return _CSTD _LDtest(&_Xx) == _FINITE;
+#else // ^^^ 80-bit long double (not supported by MSVC in general, see GH-1316) / 64-bit long double vvv
     using _Traits = _Floating_type_traits<_Ty>;
     return _Float_abs_bits(_Xx) < _Traits::_Shifted_exponent_mask;
+#endif // ^^^ 64-bit long double ^^^

```

Comment:
With the `#endif` you do not need the `^^^`

Reply:
With else we conventionally use `^^^`. https://github.com/microsoft/STL/issues/351#issuecomment-564181100

Reply:
Thanks for the reviews. Done.

Reply:
urgh sorry, I should not browse issues while having a cold